config-test.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  connection: {
+    driver: "pg",
+    user: "postgres",
+    host: "0.0.0.0",
```

Comment:
answered in slack: just switched us over to `.js` to confirm this was supported appropriately, no need to actually pull in an environment variable in this demo file.